operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -595,3 +595,25 @@ Params:
 {{- define "mimir.var_dump" -}}
 {{- . | mustToPrettyJson | printf "\nThe JSON output of the dumped var is: \n%s" | fail }}
 {{- end -}}
+
+
+{{/*
+siToBytes is used to convert Kubernetes byte units to bytes.
+Only works for limited set of SI prefixes: Ki, Mi, Gi, Ti.
+
+mimir.siToBytes takes 1 argument
+  .value = the input value with SI unit
+*/}}
+{{- define "mimir.siToBytes" -}}
+    {{- if (hasSuffix "Ki" .value) -}}
+        {{- trimSuffix "Ki" .value | float64 | mul 1024 -}}
```

Comment:
Does this also round the values?

Reply:
It doesn't. Added in https://github.com/grafana/mimir/pull/4971/commits/a079a13030c66831fdc1e246cb5eb5b87b5d4c77.